### DISCUSSION TITLE
12.0.0 performance

### DISCUSSION THREAD

**Discussion by softlion** [realm/realm-dotnet#3581] [General]
I wanted to say GOOD WORK on 12.0  
Performance is there.  
And it's still working as advertised.

Ty for your hard work, it's paying !

Btw everyone moved to net8, why stay on net7 ?

**Comment by nirinchev** [comment]
Hey, thank you so much for the nice words! The Realm SDK attempts to target the lowest possible TFM to ensure broadest compatibility - even if most companies have upgraded to .NET 8, we don't want to cut off access for people who haven't gotten around to it. At the same time, you should be able to take advantage of all the improvements in the runtime, even if the TFM is .NET 7, so the only drawback of targeting a lower .NET version is that we don't have access to all the cool language API when developing Realm. This should only affect us though and not be something that impacts you as a consumer, unless I'm missing something, in which case I'd be very curious to explore how we can improve there.
